frontend/talentsearch/src/js/components/browse/PoolCard.tsx

```diff
@@ -0,0 +1,179 @@
+import { Link } from "@common/components";
+import { getPoolStream } from "@common/constants/localizedConstants";
+import React from "react";
+import { useIntl } from "react-intl";
+import Chip, { Chips } from "@common/components/Chip";
+import { getLocale, getLocalizedName } from "@common/helpers/localize";
+import { formattedDateMonthDayYear } from "@common/helpers/dateUtils";
+import { PoolAdvertisement } from "../../api/generated";
+import { useDirectIntakeRoutes } from "../../directIntakeRoutes";
+
+export interface CardProps {
+  pool: PoolAdvertisement;
+}
+
+const PoolCard = ({ pool }: CardProps & React.HTMLProps<HTMLDivElement>) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useDirectIntakeRoutes();
+
+  const poolClassificationString = (): string | null => {
+    if (pool.classifications && pool.classifications[0]) {
+      const classification = pool.classifications[0];
+      return `${classification.group}-0${classification.level}`;
+    }
+    return null;
+  };
+
+  const poolClassificationSalary = (): string | null => {
+    if (pool.classifications && pool.classifications[0]) {
+      const classification = pool.classifications[0];
+      if (!classification.minSalary && !classification.maxSalary) {
+        return null;
+      }
+      if (classification.minSalary && !classification.maxSalary) {
+        return `$${classification.minSalary} -`;
+      }
+      if (!classification.minSalary && classification.maxSalary) {
+        return `- $${classification.maxSalary}`;
+      }
+      return `$${classification.minSalary} - $${classification.maxSalary}`;
+    }
+    return null;
+  };
```

Comment:
## Tip

You could use the helper to get the formatted salary which also does some localization for us as well.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/855abba85d6de4d4197c4a0984f2adcecfe64901/frontend/common/src/helpers/localize.ts#L84-L106

Reply:
yep https://github.com/GCTC-NTGC/gc-digital-talent/pull/4233/commits/74bd2f4fbb08dd759ac22a492198f2e6b19bb196
need to read through all the helpers/util stuff at some point